recipes/openpiv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.20.4" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  fn: openpiv-{{ version }}.tar.gz
+  url: https://github.com/OpenPIV/openpiv-python/archive/{{ version }}.tar.gz
+  sha256: 4921c30d5f6ee5234ba69ec09ab4e2e6a4913a5a3ac99c8e6bd1a09e7ba7e9ba
+  
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.6*|2.7*
+    - setuptools
+    - numpy
+    - cython
+
+  run:
+    - python 2.6*|2.7*
```

Comment:
Please drop the version constraint as it is handled by the `skip` suggested above.